src/gameplay/cards/EnvironmentCard.js

```diff
@@ -0,0 +1,18 @@
+const Card = require('./Card');
```

Comment:
This class does not seem to add any functionality to the base class, might be better to have an enum to describe the card types rather than just a new subclass that adds nothing new.